data/WAH_2022_1/metadata.yml

```diff
@@ -0,0 +1,3171 @@
+source:
+  primary:
+    key: WAH_2022_1
+    bibtype: Online
+    year: 2022
+    author: unknown
+    title: Florabase
+    url: https://florabase.dpaw.wa.gov.au/
+contributors:
+  data_collectors:
+  - last_name: Paczkowska
+    given_name: Grazyna
+    ORCID: .na.character
+    affiliation: Western Australian Herbarium, Keiran McNamara Conservation Science
+      Centre, Department of Biodiversity, Conservation and Attractions, Western Australia,
+      Australia
+  - last_name: Chapman
+    given_name: Alex
+    ORCID: .na.character
+    affiliation: Western Australian Herbarium, Keiran McNamara Conservation Science
+      Centre, Department of Biodiversity, Conservation and Attractions, Western Australia,
+      Australia
+  - last_name: Huisman
+    given_name: John
+    ORCID: 0000-0002-5255-8423
+    affiliation: Western Australian Herbarium, Biodiversity and Conservation Science,
+      Department of Biodiversity, Conservation and Attractions, Western Australia,
+      Australia
+  - last_name: Richardson
+    given_name: Ben
+    ORCID: 0000-0002-6108-0572
+    affiliation: Western Australian Herbarium, Department of Biodiversity, Conservation
+      and Attractions, Western Australia, Australia
+  - last_name: Spooner
+    given_name: Amanda
+    ORCID: .na.character
+    affiliation: Western Australian Herbarium, Keiran McNamara Conservation Science
+      Centre, Department of Biodiversity, Conservation and Attractions, Western Australia,
+      Australia
+    notes: no contact
+  - last_name: Coleman
+    given_name: Helen
+    ORCID: .na.character
+    affiliation: Western Australian Herbarium, Keiran McNamara Conservation Science
+      Centre, Department of Biodiversity, Conservation and Attractions, Western Australia,
+      Australia
+  - last_name: Coleman
+    given_name: David
+    ORCID: 0000-0002-1773-6597
+    affiliation: Evolution & Ecology Research Centre, School of Biological, Earth,
+      and Environmental Sciences, UNSW Sydney, Australia
+    notes: Responsible for extracting data from online resource
+    additional_role: contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      filter(str_detect(taxon_name, " "))
```

Comment:
Definitely needed?

Reply:
Yes. To filter out genus and family-level values since AusTraits is only meant to include traits at the species-level.